dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -779,12 +786,14 @@ public Experiment deleteTargetingCondition(String experimentId, String condition
 
     }
 
-    @CloseDBIfOpened
     @Override
     public List<Experiment> getRunningExperiments() throws DotDataException {
-        return FactoryLocator.getExperimentsFactory().list(
-                ExperimentFilter.builder().statuses(set(Status.RUNNING)).build()
-        );
+        final List<Experiment> cached = experimentsCache.get();
+        if (Objects.nonNull(cached)) {
```

Comment:
Experiments in different states can be put in the cache, so we need to filter here by RUNNING ones only.

Reply:
This differs from what @freddyucv requested. I guess you @dsilvam are right.

Reply:
Done.